deploy/eck-elasticsearch/values.yaml

```
@@ -85,11 +137,160 @@ nodeSets:
     #
     node.store.allow_mmap: false
   podTemplate:
+    # The following spec is exactly the Kubernetes Core V1 PodTemplateSpec. Any fields within the PodTemplateSpec
+    # are supported within the 'spec' field below.  Please see below documentation for the exhaustive list of fields.
+    #
+    # https://v1-24.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#podtemplatespec-v1-core
+    #
+    # Only the commonly overridden/used fields will be noted below.
+    # 
     spec:
+
+    # If specified, the pod's scheduling constraints
+    # https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-advanced-node-scheduling.html
+    # https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/
+    # affinity:
+    #   nodeAffinity:
+    #     requiredDuringSchedulingIgnoredDuringExecution:
+    #       nodeSelectorTerms:
+    #       - matchExpressions:
+    #         - key: topology.kubernetes.io/zone
+    #           operator: In
+    #           values:
+    #           - antarctica-east1
+    #           - antarctica-west1
+
+    # Containers array.  Should only be used to customize the 'elasticsearch' container using the following fields.
       containers:
       - name: elasticsearch
+
+        # List of environment variables to set in the 'elasticsearch' container.
+        # https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/
+        # env:
+        # name/value environment variable pair to set within the 'elasticsearch' container.
+        # - name: "my-env-var"
+        #   value: "my-value"
+
+        # Compute Resources required by this container.
         resources:
-          requests:
-            cpu: 1
+          # Requests describes the minimum amount of compute resources required. If Requests is omitted for a container,
+          # it defaults to Limits if that is explicitly specified, otherwise to an implementation-defined value.
+          # 
+          # Defaults used by the ECK Operator, if not specified, are below
           limits:
-            memory: 1Gi
+            # cpu: 1
+            memory: 2Gi
+          requests:
+            # cpu: 1
+            memory: 2Gi
+
+          # Example increasing both the requests and limits values:
+          # limits:
+          #   cpu: 4
+          #   memory: 8Gi
+          # requests:
+          #   cpu: 1
+          #   memory: 4Gi
```

```suggestion
          #   memory: 8Gi
```
I'd follow to the best practice memory limit=request.